src/NServiceBus.AcceptanceTests/Routing/When_sending_non_message_with_routing_configured_via_mappings.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace NServiceBus.AcceptanceTests.BestPractices
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_sending_non_message_with_routing_configured_via_mappings : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_throw_when_sending()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Receiver>()
```

Comment:
We only need the type for the `AddMapping` to work but I think we don't need to start it.